src/lib/autocomplete/BUILD.bazel

```diff
@@ -0,0 +1,40 @@
+package(default_visibility=["//visibility:public"])
+load("@angular//:index.bzl", "ng_module")
+load("@io_bazel_rules_sass//sass:sass.bzl", "sass_library")
+load("@io_bazel_rules_sass//sass:sass.bzl", "sass_binary")
+
+
+ng_module(
+  name = "autocomplete",
+  srcs = glob(["**/*.ts"], exclude=["**/*.spec.ts"]),
+  module_name = "@angular/material/autocomplete",
+  assets = [":autocomplete_css"],
+  deps = [
+    "//src/lib/core",
+    "//src/lib/form-field",
+    "//src/cdk/a11y",
+    "//src/cdk/bidi",
+    "//src/cdk/collections",
+    "//src/cdk/keycodes",
+    "//src/cdk/portal",
+    "//src/cdk/overlay",
+  ],
+  tsconfig = ":tsconfig-build.json",
+)
+
+
```

Comment:
buildozer is available externally, you could format all your BUILD files

Reply:
Ack, I'll pursue that in a follow-up.